app/views/refinery/authentication/devise/admin/users/_actions.html.erb

```diff
@@ -1,7 +1,5 @@
-<ul>
-  <li><%= render '/refinery/admin/search', :url => refinery.authentication_devise_admin_users_path -%></li>
-  <li>
-    <%= link_to t('.create_new_user'),
-                refinery.new_authentication_devise_admin_user_path, :class => "add_icon" %>
-  </li>
-</ul>
+<div class="contextual-menu">
+  <%= link_to refinery.new_authentication_devise_admin_user_path, class: "btn btn-primary" do %>
+    <i class="icon icon-add"></i> <span class="hidden-xs-down"><strong><%= t('.create_new_user') %></strong></span>
```

Comment:
This icon use the old way in regards of https://github.com/refinery/refinerycms/pull/3201